sbc_translation/wizards/reply_to_comments.py

```diff
@@ -0,0 +1,14 @@
+from odoo import models, fields, api
+
+
+class ReplyToComments(models.TransientModel):
+    _name = "reply.to.comments"
+
+    paragraph_ids = fields.Many2many("correspondence.paragraph", string="Paragraphs", readonly=True)
+    answer = fields.Text(string="Answer")
+
+    @api.multi
+    def send_reply(self):
+        corr = self.env['correspondence'].search([('id', '=', self._context['active_id'])])
```

Comment:
Browse method is designed for this:
```suggestion
        corr = self.env["correspondence"].browse(self.env.context.get("active_id"))
```